src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportService;
+import java.util.List;
+
+public class ReportServiceImpl implements ReportService {
+    private final FruitDao fruitDao;
+
+    public ReportServiceImpl(FruitDao fruitDao) {
+        this.fruitDao = fruitDao;
+    }
+
+    @Override
+    public String getReport() {
```

Comment:
The solution is ok
But you can consider making this service less dependent on Dao. What about creating a method in FruitShopService that can return a Map<String, Integer> with all the data? And passing the Map to this method as a parameter?